tests/test_markup.py

```diff
@@ -177,6 +177,28 @@ def test_load_system(
         assert value in entity.entity.value["value"]
 
 
+@pytest.mark.load
+@pytest.mark.system
+@pytest.mark.parametrize(
+    "query, expected_values",
+    [("let's meet {after 3:00|sys_interval}", [('2020-06-12T15:00:00.000-07:00', None),
+                                               ('2020-06-13T03:00:00.000-07:00', None),
+                                               ('2020-06-13T15:00:00.000-07:00', None)]),
+     ("let's meet {until 3:00|sys_interval}",
```

Comment:
"let's meet until 3" to avoid the weird bounded interval result

Reply:
done